src/framework/extensions/internal/extensionsloader.cpp

```diff
@@ -36,14 +36,14 @@ using namespace muse::extensions;
 const std::string MANIFEST("manifest.json");
 const std::string DEV_EXTENSIONS("extensions/dev/");
 
-ManifestList ExtensionsLoader::loadManifesList(const io::path_t& defPath, const io::path_t& extPath) const
+ManifestList ExtensionsLoader::loadManifestList(const io::path_t& defPath, const io::path_t& extPath) const
```

Comment:
just a spelling fix, this code isn't actually being used for loading plugins currently AFAICT